.github/workflows/adhoc.yml

```
@@ -76,47 +47,34 @@
-          username: guarzo
-          password: ${{ secrets.DOCKERHUB_PAT }}
-
       - name: Build Docker image (SHA tagged)
-        uses: docker/build-push-action@v5
+        uses: ./.github/actions/common-docker-build
         with:
-          context: .
-          push: false
-          load: true
+          push: "false"
           tags: guarzo/wanderer-notifier:sha-${{ steps.version.outputs.short_sha }}
```

actionlint (1.7.4)

property "short_sha" is not defined in object type {}

(expression)